src/components/organisms/LogoBox.jsx

```diff
@@ -76,27 +141,65 @@ const LogoContainer = styled.div`
     }
   }
 
-  &.bottom-right {
+  &bottom-right {
     align-items: flex-end;
     justify-content: flex-end;
 
     & > div > div {
       justify-content: flex-end;
     }
   }
+}`
+
+const LogoContainer = styled.div`
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  display: flex;
+  margin: var(--margin-logo);
+  z-index: ${({ zIndex }) => zIndex};
+  pointer-events: none;
+
+  ${stylesLogoDesktop}
+
+  ${({ isDifferentPositions }) =>
+    isDifferentPositions &&
+    css`
+      @media ${MediaMobile} {
+        ${stylesLogoMobile}
+      }
+
+      ${({ isPreviewMobile }) =>
+        isDifferentPositions &&
```

Comment:
isDifferentPositions is not necessary a second time. It has already been checked above ;)